tfhe/examples/dark_market/main.rs

```diff
@@ -0,0 +1,111 @@
+use std::time::Instant;
```

Comment:
dark market changes are just file reorgs ?

Reply:
did you check the updated function works as expected ?

Reply:
Yes, I just had to fix an underflow which got caught in devo profile